front/src/modules/relation-picker/components/SingleEntitySelect.tsx

```diff
@@ -97,7 +100,9 @@ export function SingleEntitySelect<
           </DropdownMenuSelectableItem>
         ))}
         {entitiesInDropdown?.length === 0 && (
-          <DropdownMenuItem>No result</DropdownMenuItem>
+          <DropdownMenuItem>
```

Comment:
this should not replace the "No result" section.
There are actual cases when data is loaded but there is no result